packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -92,6 +92,7 @@ export type ViewConfig = $ReadOnly<{
 export type PartialViewConfig = $ReadOnly<{
   bubblingEventTypes?: $PropertyType<ViewConfig, 'bubblingEventTypes'>,
   directEventTypes?: $PropertyType<ViewConfig, 'directEventTypes'>,
+  supportsRawText?: boolean,
```

Comment:
Looks like this should also be added to the `ViewConfig` type - possibly as optional for now, then required when we've updated React Native to always supply it.